/dwarn java.lang.ArrayIndexOutOfBoundsException

If you write **/dwarn**, without any args, it gives this error:

```
[17:58:20] [Server thread/WARN]: Unexpected exception while parsing console command "dwarn"
org.bukkit.command.CommandException: Unhandled exception executing command 'dwarn' in plugin BanManager v4.0
    at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[spigot.jar:git-Spigot-1370]
    at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:180) ~[spigot.jar:git-Spigot-1370]
    at org.bukkit.craftbukkit.v1_7_R2.CraftServer.dispatchCommand(CraftServer.java:719) ~[spigot.jar:git-Spigot-1370]
    at org.bukkit.craftbukkit.v1_7_R2.CraftServer.dispatchServerCommand(CraftServer.java:706) [spigot.jar:git-Spigot-1370]
    at net.minecraft.server.v1_7_R2.DedicatedServer.ax(DedicatedServer.java:309) [spigot.jar:git-Spigot-1370]
    at net.minecraft.server.v1_7_R2.DedicatedServer.v(DedicatedServer.java:274) [spigot.jar:git-Spigot-1370]
    at net.minecraft.server.v1_7_R2.MinecraftServer.u(MinecraftServer.java:566) [spigot.jar:git-Spigot-1370]
    at net.minecraft.server.v1_7_R2.MinecraftServer.run(MinecraftServer.java:472) [spigot.jar:git-Spigot-1370]
    at net.minecraft.server.v1_7_R2.ThreadServerApplication.run(SourceFile:618) [spigot.jar:git-Spigot-1370]
Caused by: java.lang.ArrayIndexOutOfBoundsException: 0
    at me.confuser.banmanager.commands.DeleteLastWarningCommand.onCommand(DeleteLastWarningCommand.java:33) ~[?:?]
    at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[spigot.jar:git-Spigot-1370]
    ... 8 more
```

--- a/src/me/confuser/banmanager/commands/DeleteLastWarningCommand.java
+++ b/src/me/confuser/banmanager/commands/DeleteLastWarningCommand.java
@@ -19,6 +19,9 @@ public class DeleteLastWarningCommand implements CommandExecutor {
 	}
 
 	public boolean onCommand(CommandSender sender, Command command, String commandLabel, String args[]) {
+		if(args.length != 1)
+			return false;
+
 		Player player = null;
 
 		if (sender instanceof Player) {

